src/BaseBallGame.js

```diff
@@ -0,0 +1,44 @@
+export default class BaseballGame {
+  play(computerInputNumbers, userInputNumbers){
+    let strikeCount = 0;
+    let ballCount = 0;
+
+    for(let i = 0 ; i < 3; i+=1){
+      let result = this.compare(computerInputNumbers,userInputNumbers[i], i);
+      if(result === 'strike'){
+        strikeCount+=1;
+      }
+      else if(result === 'ball'){
+        ballCount+=1;
+      }
```

Comment:
```suggestion
      if(result === 'strike'){
        strikeCount+=1;
      }
      if(result === 'ball'){
        ballCount+=1;
      }
```
이렇게 사용하는 것이 더 좋아보이는데 건님은 어떻게 생각하시나요? 왜 if-else if를 사용하였는지 공유해주시면 같이 이야기 나누어보아도 좋을 것 같아요

Reply:
위 if 문은 결과가 strike 인지 ball인지 아니면 그 외 값 인지에 따라 서로 다른 코드를 실행시켜야 하는 구문입니다. 
else if 대신 if를 사용하여도 코드 작동에는 문제가 없지만, if else를 사용하는 것이 좀 더 의미에 적합하다고 생각하였습니다!